api/v1alpha1/contentlibraryitemimportrequest_types.go

```diff
@@ -0,0 +1,164 @@
+// Copyright (c) 2023 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ContentLibraryItemImportRequestSource contains the specification of the source for the import request.
+type ContentLibraryItemImportRequestSource struct {
+	// URL is the endpoint that points to an OVF/OVA template that
+	// has to be imported as a new Content Library Item in the
+	// vSphere Content Library.
+	// +required
+	URL string `json:"url"`
+}
+
+// ContentLibraryItemImportRequestTargetItem contains the specification of the target
+// content library item for the import request.
+type ContentLibraryItemImportRequestTargetItem struct {
+	// Name is the name of the new content library item that will be created in vSphere.
+	// If omitted, the content library item will be created with the same name as the name
+	// of the image specified in the spec.source.url, only if an item with the same name does not exist
```

Comment:
> only if an item with the same name does not exist
If an item with the same name already exists in the specified Content Library, the TargetValid condition will become false in the status.

Reply:
Updated.